.editorconfig

```diff
@@ -0,0 +1,27 @@
+; EditorConfig to support per-solution formatting.
```

Comment:
This is actually a new VS2017 thing. cref https://blogs.msdn.microsoft.com/dotnet/2016/12/15/code-style-configuration-in-the-vs2017-rc-update/